README.md

```diff
@@ -217,6 +252,46 @@ Each cluster that will be connected must have Submariner installed within it. Yo
    |\<SERVICE_CIDR>|Service CIDR for Cluster|""|`10.43.0.0/16`|
    |\<NAT_ENABLED>|If in a cloud provider that uses 1:1 NAT between instances (for example, AWS VPC), you should set this to `true` so that Submariner is aware of the 1:1 NAT condition.|"false"|`false`|
 
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md).
+
+### Generating the Operator
+
+The current (developer-oriented) implementation dynamically generates the
+operator. This allows us to consume updates to the underlying best practices of
+the Operator SDK. It also results in a clear, working example of how to use the
+Operator SDK to create additional operators (perhaps for future parts of
+Submariner).
+
+```
+  cd operators/go
+  make codegen-operator
+```
+
+That will run the operator sourcecode generation logic in ./gen_subm_operator.sh
+
+### Builiding the operator
+
+```
+cd operators/go
+make build-operator
+```
+
+### Deploying Submariner using the Operator
+
+After generating the Operator (see docs above), your newly generated operator
+is automatically fully integrated into the Submariner CI automation. Simply use
+the `deploytool` flag to the standard `make` commands.
+
+```make e2e status=keep deploytool=operator```
+
+A large set of verifications for the Operator and the resulting Submariner
+deployment will automatically run during and after the deployment.
```

Comment:
run->ran?

Reply:
run is appropriate becasue of 'will'. Its like once you run this command, tests will run